docs/installation/index.md

```diff
@@ -5,10 +5,22 @@ sidebar: sidebars/documentation.html
 script: osmenu.js
 ---
 
+{% if site.version == null %}
+# Katello Nightly Installation
+{% else %}
 # Katello Installation
+{% endif %}
+
+{{ site.version }}
+
+{% if site.version == null %}
+  <div class="alert alert-danger">
+    These are the instructions for installing the unstable nightly release of katello!
```

Comment:
Any thoughts on this? I think it'd go well with your RC change, that way the RC gets the same kind of warning.